Makefile

```diff
@@ -1,336 +1,44 @@
-	echo -> $@
-
-dist :
-	git worktree add $@ $(meta_distBranch)
-
-$(wildcard dist/* dist/*/*) : dist
-	@
-
-distready : dist $(wildcard dist/* dist/*/*)
-	cd dist && git checkout $(meta_distBranch)
-	cd dist && git pull
-
-.events2/push-git : .git/refs/heads .git/refs/tags $(wildcard .git/refs/heads/* .git/refs/tags/*) | .events2 distready
-	git push origin --tags -f
-	git push origin --all
-	echo -> $@
-
-.events2/push-web : .git/refs/heads/$(meta_distBranch) | .events2 distready
-	git push web --tags -f
-	git push web $(meta_distBranch)
-	echo -> $@
-
-.events2/push-store : .git/refs/tags/stable | .events2 distready
-	node tools/webstore.js
-	echo -> $@
-
-.SECONDARY :
-
-.PHONY: default all distready script crx release jshint install push $(npgoals)
-
-script : $(script)
-
-crx : $(crx)
-
-release : $(release)
-
-jshint : .events/jshint
-
-install : .events/install
-
-push : .events2/push-git .events2/push-web .events2/push-store
-
-clean :
-	$(RMDIR) tmp tmp-crx testbuilds .events
-	$(RM) .tests_enabled
-
-cleanrel : clean
-	$(RMDIR) builds
-
-cleanweb :
-	$(RM) test.html
-
-cleanfull : clean cleanweb
-	$(RMDIR) .events2 dist node_modules
-	git worktree prune
-
-withtests :
-	echo true> .tests_enabled
-	-$(MAKE)
-	echo false> .tests_enabled
-
-archives :
-	git fetch -n archives
-	git merge --no-commit -s ours archives/gh-pages
-	git show archives/gh-pages:archives.json > src/Archive/archives.json
-	-git commit -am 'Update archive list.'
-
-$(foreach i,1 2 3 4,bump$(i)) :
-	$(MAKE) archives
-	node tools/bump.js $(subst bump,,$@)
-	$(MAKE) .events/CHANGELOG
-	$(MAKE) all
-
-tag :
-	git add builds
-	$(MAKE) cleanrel
-	$(MAKE) all
-	git diff --quiet -- builds
-	$(MAKE) tagcommit
-
-tagcommit :
-	git commit -am "Release $(meta_name) v$(version)."
-	git tag -a $(version) -m "$(meta_name) v$(version)."
-
-beta : distready
-	git tag -af beta -m "$(meta_name) v$(version)."
-	cd dist && git merge --no-commit -s ours beta
-	cd dist && git checkout beta "builds/*-beta.*" img .gitignore .gitattributes
-	cd dist && git commit -am "Move $(meta_name) v$(version) to beta channel."
-
-stable : distready
-	git push . HEAD:bstable
-	git tag -af stable -m "$(meta_name) v$(version)."
-	cd dist && git merge --no-commit -s ours stable
-	cd dist && git checkout stable "builds/$(name).*" builds/updates.xml builds/updates.json
-	cd dist && git commit -am "Move $(meta_name) v$(version) to stable channel."
-
-web : index.html distready
-	-git commit -am "Build web page."
-	cd dist && git merge --no-commit -s ours master
-	cd dist && git checkout master README.md index.html web.css img .gitignore .gitattributes
-	cd dist && git commit -am "Update web page."
-
-update :
-	$(RM) package-lock.json
+# Updates npm packages
+update:
+	@echo "Updating the repository..."
 	npm install --save-dev $(shell node tools/unpinned.js)
 	npm install
-
-updatehard :
-	$(RM) package-lock.json
-	npm install --save-dev $(shell node tools/unpinned.js latest)
-	npm install
```

Comment:
Sorry mate, but I can't merge a PR that removes functionality without providing an alternative. Like mentioned in https://github.com/ccd0/4chan-x/pull/3341#issuecomment-1456805345, this makefile also including code to update their webpage, which I don't want do delete in my upstream PR.

Also, the few files that weren't affected by the migration around is also something I don't want in the upstream PR.

Reply:
Understandable, I'm changing back to only bug fixes and so on. My idea was that it would be rewritten or added on if it still was needed due to all of the bloat that i felt was in there, I will see if i can revert the commits